src/vellum/workflows/nodes/displayable/bases/api_node/node.py

```diff
@@ -83,9 +84,13 @@ def _local_execute_api(self, data, headers, json, method, url):
 
     def _vellum_execute_api(self, bearer_token, data, headers, method, url):
         client_vellum_secret = ClientVellumSecret(name=bearer_token.name) if bearer_token else None
-        vellum_response = self._context.vellum_client.execute_api(
-            url=url, method=method.value, body=data, headers=headers, bearer_token=client_vellum_secret
-        )
+        try:
+            vellum_response = self._context.vellum_client.execute_api(
+                url=url, method=method.value, body=data, headers=headers, bearer_token=client_vellum_secret
+            )
+        except ApiError as e:
+            NodeException(f"Failed to prepare HTTP request: {e}", code=WorkflowErrorCode.NODE_EXECUTION)
```

Comment:
[blocking] we're missing the `raise` here. let's fast follow with a test that would've caught this :D